### DISCUSSION TITLE
Monitor Flows

### DISCUSSION THREAD

**Discussion by brotoo25** [postmanlabs/postman-flows#175] [Ideas]
It'd be great to Monitor Flows exactly the same way we currently monitor Collections in Postman. Is it in the roadmap?

**Comment by ckang777** [comment]
We would like this feature as well.

**Comment by TomVanCamping** [comment]
Enormously powerful feature! Would really love to see it!

**Comment by viko42** [comment]
It would be a very good feature

**Comment by tom-mrnk-github** [comment]
🙋‍♂️ would love to see it too. Is there any feedback from Postman product management? The idea was brought up nearly a year ago...
